src/Microsoft.DotNet.Cli.Utils/CommandResolution/OutputPathCommandResolver.cs

```diff
@@ -80,7 +80,7 @@ private ProjectContext GetProjectContextFromDirectory(string directory, NuGetFra
             var projectContext = ProjectContext.Create(
                 projectRootPath,
                 framework,
-                DotnetRuntimeIdentifiers.InferCurrentRuntimeIdentifiers());
+                RuntimeEnvironmentRidExtensions.GetAllCandidateRuntimeIdentifiers());
```

Comment:
1. This code should be going away.
2. This code can't guarantee that it is running in the context of a CLI, so there is no way to get to the CLI's .version file.  Thus, no way to RID cap.